angular-ngrx-scss/src/app/repository/services/repository.service.ts

```diff
@@ -47,6 +52,40 @@ export class RepositoryService {
     return this.http.get<[]>(url).pipe(map((data) => data.length));
   }
 
+  getRepositoryPullRequest(
```

Comment:
question / issue: Do we have a function for pulling **all** the pull requests for a repo? Or only this function for getting info on a single PR? 

I believe what we need for this ticket is to get info on all the PRs for the requested repo. Unless there's some context I'm missing on how this will get called?

Reply:
Oh sorry, I missed the plural part. I can add the methods for getting multiple as well.